src/test/scala/org.openeo/OpenSearchClientTest.scala

```diff
@@ -198,5 +243,155 @@ class OpenSearchClientTest {
     assertEquals(features.size,unique.size)
   }
 
+  @ParameterizedTest
+  @MethodSource(Array("level1CParams"))
+  def testManifestLevelSentinel2_L1C(date: LocalDate, processingBaseline: java.lang.Double): Unit = {
+    // Cache reduces test time from 5min to 1sec.
+    val httpsCacheEnabledOriginalValue = HttpCache.enabled
```

Comment:
Same as before, best to avoid the try finally structure by using BeforeClass and AfterClass.

Reply:
ok, done